config.go

```diff
@@ -17,8 +17,7 @@ type Config struct { // nolint: aligncheck
 	Linters map[string]string
 
 	// The set of linters that should be enabled.
-	Enable  []string
-	Disable []string
```

Comment:
This can not be removed. It is for users to disable already enabled linters. For example, if they want to use the default set of linters, except `staticcheck`, they would set `"Disable": ["staticcheck"]`